src/_guides/whats-new.md

```diff
@@ -17,6 +17,17 @@ and follow the [Dart blog][].
 [dart-announce]: https://groups.google.com/a/dartlang.org/d/forum/announce
 [Dart blog]: https://medium.com/dartlang
 
+## August 30, 2022: 2.18 release
+
+This section lists notable changes made from May 12, 2022,
+through August 30, 2022.
+For details about the 2.18 release,
+see [Dart 2.18: ][], and the [change log][https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#2180].
+
+In addition to bug fixes and incremental improvements,
+we made the following changes to this site:
+
```

Comment:
is your intent to also add here some bullet points about these recent changes?

Reply:
I guess this was accidentally included as that is being handled in https://github.com/dart-lang/site-www/pull/4195